apps/labs/posts/ibis-python-data-analysis-productivity-framework.md

```diff
@@ -0,0 +1,248 @@
+relational database designed from the ground up to run on GPUs. If you
+don't know yet this amazing database, I invite you to [check it
+out](https://omnisci.com).
+
+The implementation of this new backend also resulted in the creation of
+new expressions/operators on Ibis core, such as:
+
+-   GeoSpatial data types and operations
+-   Trigonometric operations
+-   Some statistcal operations
+
+First, let's connect to a *OmniSciDB* and play with this new features!
+
+``` python
+# install the dependencies if you need!
+# !conda install -y ibis-framework=1.1.0 pyarrow pymapd vega geopandas geoalchemy2 shapely matplotlib --force-reinstall
+```
+
+``` python
+import ibis
+from matplotlib import pyplot as plt
+
+print('ibis:', ibis.__version__)
+```
+
+    ibis: 1.2.0+7.g3afa8b0d
+
+``` python
+# metis.mapd.com is used in some OmniSci docs
+# but maybe you want to install your own OmniSciDB instance
+# you can take a look into installation section at 
+# https://www.omnisci.com/docs/latest/
+# also you maybe want to check the omniscidb-cpu conda package
+# conda install -c conda-forge omniscidb-cpu
+# if you need any help, feel free to open an issue at
+# https://github.com/conda-forge/omniscidb-cpu-feedstock/
+omniscidb_cli = ibis.mapd.connect(
+    host='metis.mapd.com', 
+    user='mapd', 
+    password='HyperInteractive',
+    port=443, 
+    database='mapd',
+    protocol='https'
+)
+```
+
+### GeoSpatial features
+
+You need to handle geospatial data in a esier way?
+
+Let's take a look inside `zipcodes_2017` table!
+
+Well, currently `omniscidb` backend doesn't support `geopandas` output,
+so let's use a workaround for that! It should be implemented into
+`omniscidb` backend soon! (see:
+[gist-code](https://gist.githubusercontent.com/xmnlab/587dd1bde44850f3117a1087ed3f0f28/raw/0750400db90cf97319a91aa514648c31ad4ace45/omniscidb_geopandas_output.py))
+
+``` python
+gist_url = 'https://gist.githubusercontent.com/xmnlab/587dd1bde44850f3117a1087ed3f0f28/raw/0750400db90cf97319a91aa514648c31ad4ace45/omniscidb_geopandas_output.py'
+!wget {gist_url} -O omniscidb_geopandas_output.py
+```
+
+    --2019-07-05 11:31:57--  [https://gist.githubusercontent.com/xmnlab/587dd1bde44850f3117a1087ed3f0f28/raw/0750400
+    db90cf97319a91aa514648c31ad4ace45/omniscidb_geopandas_output.py](https://gist.githubusercontent.com/xmnlab/587dd1bde44850f3117a1087ed3f0f28/raw/0750400db90cf97319a91aa514648c31ad4ace45/omniscidb_geopandas_output.py)
+    Resolviendo gist.githubusercontent.com (gist.githubusercontent.com)... 151.101.48.133
+    Conectando con gist.githubusercontent.com (gist.githubusercontent.com)[151.101.48.133]:443... conectado.
+    Petición HTTP enviada, esperando respuesta... 200 OK
+    Longitud: 1874 (1,8K) [text/plain]
+    Guardando como: “omniscidb_geopandas_output.py”
+
+    omniscidb_geopandas 100%[===================>]   1,83K  --.-KB/s    en 0s      
+
+    2019-07-05 11:31:57 (70,1 MB/s) - “omniscidb_geopandas_output.py” guardado [1874/1874]
```

Comment:
Our MDX compiler does not recognize indented text as a code block, only [fenced](https://spec.commonmark.org/0.30/#fenced-code-blocks) text.